tools/install_prestashop.sh

```diff
@@ -0,0 +1,9 @@
+
+echo Clone Prestashop
+   
+# Clone Prestashop into Build Folder
+cd $TRAVIS_BUILD_DIR
+git clone --depth=1 --branch=$PS_VERSION https://github.com/Prestashop/Prestashop.git $TRAVIS_BUILD_DIR 
+
+# PrestaShop configuration
+cp tests/parameters.yml.travis app/config/parameters.yml
```

Comment:
This is enough 🤔 ? No need to perform any more steps ? 😮

Reply:
yes!

nahhh, look at https://github.com/mickaelandrieu/coinbase-commerce-prestashop/pull/1/files#diff-d7949b08658567f93cfee17ea671b996R5

Reply:
I've renamed the script